Nare_Mkrtchyan/python/25_05/perfect.py

```diff
@@ -0,0 +1,31 @@
+def check(num):
+"""
+    Check function checks if user input is number or not. If it is number returns true, otherwise returns false.
+"""
+
+    try:
+        int(num)
+        return True
+    except ValueError:
+        return False
+
+def perf(n):
+    """
+    perf function checks if user input is perfect number or not.
+    """
+    mysum=0
+    for i in range(1,n-1):
+        if n%i==0:
+            mysum=mysum+i
+    if n==mysum:
+        print(n, "is perfect number")
+    else:
+        print(n,"is not perfect number")
+
+print("Enter number to check if it is perfect or not:")
+num=input()
+if check(num):
+    num=int(num)
+    perf(num)
+else:
+    print(num,"is not number. Enter number.")
```

Comment:
1. ![image](https://user-images.githubusercontent.com/80816144/119563852-23135100-bdb9-11eb-9f74-432cc0660964.png)

2. Perfect number is a positive integer
![image](https://user-images.githubusercontent.com/80816144/119564002-4d650e80-bdb9-11eb-9acf-dc646c1ca023.png)

3. mysum -> coding style - under_scores for variables, add spaces please

4. limitation
![image](https://user-images.githubusercontent.com/80816144/119564491-e85de880-bdb9-11eb-9db6-aebe482cf223.png)

Reply:
Tried to do marked changes.